insights/protocol.py

```diff
@@ -77,25 +80,26 @@ def get_network_id(network):
 
 def get_model_id(model_type):
     return {
-        MODEL_TYPE_FUNDS_FLOW: MODEL_TYPE_FUNDS_FLOW_ID
+        MODEL_TYPE_FUNDS_FLOW: MODEL_TYPE_FUNDS_FLOW_ID,
+        MODEL_TYPE_BALANCE_TRACKING: MODEL_TYPE_BALANCE_TRACKING_ID,
     }.get(model_type)
 
 def get_model_types():
-    return [MODEL_TYPE_FUNDS_FLOW]
+    return [MODEL_TYPE_FUNDS_FLOW, MODEL_TYPE_BALANCE_TRACKING]
 
 def get_networks():
     return [NETWORK_BITCOIN]
 
 class DiscoveryMetadata(BaseModel):
     network: str = None
-    model_type: str = None
-    graph_schema: Optional[Dict] = None
+    model_types: List[str] = None
     #TODO: implement method for getting graph schema from miner
 
 class DiscoveryOutput(BaseModel):
     metadata: DiscoveryMetadata = None
     block_height: int = None
     start_block_height: int = None
+    balance_model_last_block: int = None
```

Comment:
idea: lets add funds_flow_model_last_block and keep block_height for compatibility ; please verify if that does make sense

Reply:
Your idea is to have like the following?
```
class DiscoveryOutput(BaseModel):
    metadata: DiscoveryMetadata = None
    block_height: int = None
    funds_flow_model_last_block: int = None
    funds_flow_model_start_block: int = None
    balance_model_last_block: int = None
    run_id: str = None
    version: Optional[int] = VERSION
```

Reply:
balance_model_last_block, but where is balance_model_start_block ?

Reply:
or we assume that starting block for balance tracking will be always 1 ?